mcs/class/System.Private.CoreLib/System.Runtime.Loader/AssemblyLoadContext.cs

```diff
@@ -25,6 +26,7 @@ Assembly InternalLoadFromPath (string assemblyPath, string nativeImagePath)
 		{
 			StackCrawlMark stackMark = StackCrawlMark.LookForMyCaller;
 
+			assemblyPath = assemblyPath.Replace ('\\', Path.DirectorySeparatorChar);
```

Comment:
Could you add a comment on why is it needed?

Reply:
Some tests pass in windows paths to it, not sure how else to handle it.

Reply:
Are they windows specific? Could you share an example?

Reply:
https://github.com/dotnet/corefx/blob/master/src/System.Runtime/tests/System/Type/TypeTests.cs#L300